operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -0,0 +1,198 @@
+
+Command to tag an IMS image using craycli 
+```bash
+cray ims images update < Image id > --metadata-operation set --metadata-key < key > --metadata-value < value >
+```
+e.g.
+```bash
+cray ims images update bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb --metadata-operation set --metadata-key sbps-project --metadata-value true
+```
+Command to describe the IMS image metadata:
+```bash
+cray ims images describe < Image id > --format json
+e.g
+
+# cray ims images describe f1d6c8fe-32e2-420a-a051-377e34a8bd8a --format json
+{
+  "arch": "x86_64",
+  "created": "2024-07-24T20:59:37.469889",
+  "id": "f1d6c8fe-32e2-420a-a051-377e34a8bd8a",
+  "link": {
+    "etag": "af7e30ed3219ee4e5c227c5f734dca2d",
+    "path": "s3://boot-images/f1d6c8fe-32e2-420a-a051-377e34a8bd8a/manifest.json",
+    "type": "s3"
+  },
+  "metadata": {},
+  "name": "compute-uss-1.1.0-134-csm-1.5.x86_64"
+}
+
+Or
+
+cray ims images list --format=json | jq '.[] | select(.id == <Image id>)'
+e.g.
+
+# cray ims images list --format=json | jq '.[] | select(.id == "bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb")'                                    
+ {
+  "arch": "x86_64",
+  "created": "2024-07-18T22:05:16.565885",
+  "id": "bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb",
+  "link": {
+    "etag": "3325f830ba9ec291005a4087be4f666f",
+    "path": "s3://boot-images/bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb/manifest.json",
+    "type": "s3"
+  },
+  "metadata": {
+    "name": "secure-storage-ceph-6.1.94-x86_64.squashfs",
+    "sbps-project": "true"   <---------------- Tagged with key/value pair sbps-project/true 
+  },
+  "name": "secure-storage-ceph-6.1.94-x86_64.squashfs"
+}
+```
+
+Command to untag an IMS image using craycli
+```bash
+cray ims images update < Image id > --metadata-operation remove --metadata-key < key >
+```
+e.g.
+```bash
+cray ims images update a506a6f6-54d9-4e5a-9e8d-1fc052d62504 --metadata-operation remove --metadata-key sbps-project
+```
+This IMS image tagging is done automatically by BOS in the latest fix which is more efficient than manual way of doing using craycli. BOS tags the rootfs images when the boot of compute/UAN is triggered. But for untagging an image, we need to use above craycli command as untagging of an image is not supported by BOS.
+
+## Creation of BOS session template
+Once the node personalisation is done and GOSS tests are run sucessfully, create BOS Session Template with SBPS boot parameters. 
+
+See https://github.com/Cray-HPE/docs-csm/blob/release/1.6/operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md for more info.
+
+## Steps to boot compute/ UAN node
+
+Follow below steps in order to boot compute/ UANs.
+
+### To boot a single node:
+
+```bash 
+cray bos sessions create --template-name <Bos session template name> --operation reboot --limit < xname of the node>

```

Comment:
Remove the `bash`

- `<Bos session template name>` -> `<bos_session_template_name>`
- `< xname of the node>` -> `<xname_of_the_node>`